auth-web/tests/unit/components/DeactivateCard.spec.ts

```diff
@@ -4,6 +4,7 @@ import Vue from 'vue'
 import VueRouter from 'vue-router'
 import Vuetify from 'vuetify'
 import Vuex from 'vuex'
+import i18n from '@/plugins/i18n'
 
```

Comment:
you need to update all these tests to mock the correct function or even better would be to reinitialize i18n to go from a test file instead of mocking it out. Right now they only pass because you're mocking the `$t` which is actually incorrect since that function is never initialized in the new code (its `t` now)

Reply:
I made a mockI18n.ts file that reinitialized vuei18n but still had to install it onto a localVue instance is that the right way of doing this?